Formula/libfido2.rb

```diff
@@ -0,0 +1,90 @@
+class Libfido2 < Formula
+  desc "Provides library functionality for FIDO 2.0, including USB connections"
+  homepage "https://developers.yubico.com/libfido2/"
+  url "https://github.com/Yubico/libfido2/archive/1.3.0.tar.gz"
+  sha256 "0b2e3671c4c5d42fd5604a08e45f89f49592b97cf66d7d3bfbc7e6a4d5a0fec7"
+  depends_on "cmake" => :build
+  depends_on "mandoc" => :build
+  depends_on "libcbor"
+  depends_on "openssl@1.1"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "man_symlink_html"
+      system "make", "man_symlink"
+      system "make", "install"
+    end
+    mv prefix/"man", share/"man"
+  end
+
+  # TODO: Are there any built-in tests that we could run?
```

Comment:
since this is crypto related, `make check` would be okay

Reply:
See my reply on the PR: https://github.com/Homebrew/homebrew-core/pull/50326#issuecomment-589210694